test/core/image_processing/Jamfile

```diff
@@ -19,3 +19,4 @@ run threshold_truncate.cpp ;
 run threshold_otsu.cpp ;
 run lanczos_scaling.cpp ;
 run simple_kernels.cpp ;
+run box_filter.cpp ;
```

Comment:
@miralshah365 Sorry for incomplete previous suggestion.

This should read something like this

```
run box_filter.cpp /boost/test//boost_unit_test_framework : : : <link>shared:<define>BOOST_TEST_DYN_LINK=1 ;
```